src/ResourceManager/Compute/Commands.Compute/help/Get-AzureRmGallery.md

```diff
@@ -0,0 +1,115 @@
+---
+external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+Module Name: AzureRM.Compute
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/get-azurermgallery
+schema: 2.0.0
+---
+
+# Get-AzureRmGallery
+
+## SYNOPSIS
+Get or list galleries.
+
+## SYNTAX
+
+### DefaultParameter (Default)
+```
+Get-AzureRmGallery [[-ResourceGroupName] <String>] [[-GalleryName] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameter
+```
+Get-AzureRmGallery [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get or list galleries.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzureRmGallery -ResourceGroupName $rgname -GalleryName $galleryName
+```
+
+Get the gallery.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -GalleryName
+The name of the gallery.
+
+```yaml
+Type: System.String
+Parameter Sets: DefaultParameter
+Aliases: Name
```

Comment:
@hyonholee I think it would be better if the name of the parameter was `-Name` and the alias was to `-GalleryName`